src/rata-workspace/rata-workspace.service.ts

```diff
@@ -29,7 +30,8 @@ export class RataWorkspaceService {
     private readonly historicalRepository: RataRepository,
     @Inject(forwardRef(() => RataSummaryWorkspaceService))
     private readonly rataSummaryService: RataSummaryWorkspaceService,
-  ) {}
+  ) {
+  }
 
   async getRataById(id: string): Promise<RataDTO> {
     const result = await this.repository.findOneBy({
```

Comment:
This should be updated to accept a transaction and conditionally use it:
```suggestion
  async getRataById(id: string, trx?: EntityManager): Promise<RataDTO> {
    const result = await (trx
      ? trx.getRepository(this.repository.target)
      : this.repository
    ).findOneBy({
```